app/src/main/res/layout/remove_wallet_balance.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
All this relative layouts inside this layout make it really complex.
Are they necessary? i think we can implement everything with constraints

Reply:
Relative Layouts are "lighter" than constraints, we should use them whenever is not too complex to use them